### PR TITLE
Moving FAQ Where is the tool help? 

### PR DESCRIPTION
**Pull request by Beachyesh**
This file answers the FAQ "Where is the tool help?"

### CODE REVIEW

File: faqs/galaxy/tools_help.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Where is the tool help?
+area: tools   
+description:  Finding tool support
+box_type: tip
+layout: faq    
+contributors: [jennaj, beachyesh]
+---
```
Comment (beatrizserrano) [inline faqs/galaxy/tools_help.md:8]: I think that you need some blank lines here so that it renders correctly

File: faqs/galaxy/tools_help.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+There is documentation available on the tool form itself which mentions the following information:
+- Parameters
+- Expected format for input dataset(s)
+- Links to publications and ToolShed source repositories
+- Tool and wrapper version(s)
+- 3rd party author web sites and documentation
+
+Many tools include "most common" and "full parameter" run-time options. Scroll down on the tool form to locate:
```
Comment (beatrizserrano) [inline faqs/galaxy/tools_help.md:16]: the first sentence is not clear to me... and the second one looks like the actual only step of the FAQ

**Review by beatrizserrano — COMMENTED** [review]
@Beachyesh this PR means that we close #3322, correct?

File: faqs/galaxy/tools_help.md (hunk 2 above)
Comment (Beachyesh) [reply faqs/galaxy/tools_help.md]: Yes, I basically took it verbatim from the old FAQ, and it doesn't fit well. I removed that part and made the single step in this FAQ bold.

File: faqs/galaxy/tools_help.md (hunk 1 above)
Comment (Beachyesh) [reply faqs/galaxy/tools_help.md:8]: I added a blank line after the horizontal rule. That should suffice right?

**Comment by Beachyesh** [comment]
> @Beachyesh this PR means that we close #3322, correct?

Yess, I think we can close it.

**Comment by beatrizserrano** [comment]
> > @Beachyesh this PR means that we close #3322, correct?
> 
> Yess, I think we can close it.

great, will do! This one looks good to me, ready to merge, thanks :)

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**